packages/pwa-buildpack/lib/__tests__/cli-create-project.spec.js

```diff
@@ -44,9 +44,6 @@ test('is a yargs builder', async () => {
     );
 
     expect(output).toMatch('Create a PWA');
-
-    // throws because it wants a positional argument--just checking
-    expect(() => createProjectCliBuilder.builder(yargs)).toThrow('positional');
```

Comment:
Actually not sure about this one. Tests are failing if we keep this. Any feedback is appreciated.

Reply:
I thought it was an old obsolete test but unfortunately, it runs just fine in `develop`, so maybe something has changed in this PR:

Develop: 

![image](https://user-images.githubusercontent.com/35203638/146566466-0529f564-d960-4a74-a642-7c2b71e9c6a6.png)

Reply:
Yes, the test only fails on this branch. If I recall, if was something about `.toThrow('positional');`

Reply:
Here's the error generated when running `yarn test` : 
![image](https://user-images.githubusercontent.com/25036022/146770302-6353ffb7-3e63-4b37-8d08-782ee350b50e.png)

Reply:
@pedrochiossi 
I'm seeing this as of [version 17.0.0](https://github.com/yargs/yargs/releases/tag/v17.0.0) : 
`.positional() now allowed at root level of yargs`

Reply:
@jeremycharron I don't think this is related, I would look for something on builder function, but if seems it's not on their release notes.

Reply:
@tjwiebell has written a new test. I will get in touch with him so we can replace this one.